CrisisTextLine/Elmer
# error handling to remove churned script, new job to handle pending users

#### What's this PR do?
* added error handling to remove churned users script
* added a script that reviews pending WiW users and if it finds WiW deleted user w the same email it reactivates the deleted user and deletes the pending user object; if it finds an already active WiW user it deleted the pending user object

#### Where should the reviewer start?
* removePending.js

#### How should this be manually tested?
* add keys and config to the top of the removePending file
* comment out the cronjob
* node removePending.js in the console
* before running the job make sure that there are WiW pending users to be dealt with

#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/INT-349
#### Questions:
